django_x509/base/admin.py

```diff
@@ -59,11 +60,25 @@ class CaAdmin(BaseAdmin):
     list_filter = ['key_length', 'digest', 'created']
 
 
-class CertAdmin(BaseAdmin):
+class AbstractCertForm(forms.ModelForm):
+        OPERATION_CHOICES = (
+            ('-----', '-----'),
+            ('import', 'import'),
+            ('new', 'new')
+        )
```

Comment:
Invert the order of the two elements because creating new elements is usually the more common case.